### DISCUSSION TITLE
GitHub is detecting an incorrect license in my repository

### DISCUSSION THREAD

**Discussion by JakubMelka** [community/community#81440] [Repositories]
### Select Topic Area

Bug

### Body

Hello, I would like to ask for assistance with a problem where GitHub is detecting the wrong license for my repository at https://github.com/JakubMelka/PDF4QT. Despite having the correct LGPL-3.0 license, it also detects 'Unknown licenses found,' and I can't seem to remove this. The issue originated when the LGPL-3.0 license was in RTF format and was then converted to a text file. The original license file was deleted, but GitHub still seems to detect it.

How can I resolve this issue? I need to ensure that only the LGPL-3.0 license is displayed to users, and no other license is present. However, I don't want to perform any risky operations on this repository, such as rewriting history.

Does anyone know how to fix this?

**Comment by phattharaphorn** [comment]
If GitHub is detecting an "Unknown license" in addition to the LGPL-3.0 license in your repository, it might be due to remnants of the RTF format or other lingering files that GitHub's license detection is picking up.

Here are some steps you can follow to address this issue without performing any risky operations like rewriting history:

- Check for Lingering Files:
Ensure that there are no other files in your repository that resemble license files (e.g., LICENSE.md, LICENSE.txt, etc.). If there are, make sure they are properly formatted and only contain the LGPL-3.0 license text.

- Review .gitattributes File:
Check your .gitattributes file (if you have one) to ensure there are no configurations that might be affecting how files are detected or treated.

- Use GitHub's License Template:
GitHub provides a template for the LGPL-3.0 license. You might consider creating a new file named LICENSE in your repository and using GitHub's license template for the LGPL-3.0 license. This ensures that the license is in a recognized format that GitHub can easily detect.

- Clear Cache and Re-check:
Sometimes, GitHub's cache might be causing the issue. While there's no direct way to clear GitHub's cache, you can make a minor change to your repository (e.g., adding a space in a README file), commit the change, and then see if GitHub re-detects the license correctly.

- Reach Out to GitHub Support:
If the issue persists, consider reaching out to GitHub Support. They might be able to provide insights or solutions specific to your repository.

Remember, it's essential to maintain clarity and transparency regarding licensing in open-source projects. Ensuring that the correct license is displayed and recognized by platforms like GitHub helps in fostering trust and collaboration within the community.

**Reply by JakubMelka** [reply]
Hello, thank you for the answer. It helped to delete the license and recommit the license file.
